components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -537,12 +537,23 @@ void BraveSyncServiceImpl::RequestSyncData() {
 
   if (tools::IsTimeEmpty(last_fetch_time)) {
     SendCreateDevice();
+  }
+
+  sync_client_->SendFetchSyncDevices();
+
+  if (sync_prefs_->GetSyncDevices()->size() <= 1) {
+    // No sense to fetch or sync bookmarks when there no at least two devices
+    // in chain
+    // Set last fetch time here because we had fetched devices at least
+    sync_prefs_->SetLastFetchTime(base::Time::Now());
```

Comment:
then shouldn't we remove this line?
https://github.com/brave/brave-core/blob/c43d36471d713c2021259a6ea47810772f0b1eaa/components/brave_sync/brave_sync_service_impl.cc#L582

Reply:
never mind. this will be needed after sync chain setup